### ISSUE TITLE
[Bug] Getting Stuck after one failed download

### ISSUE THREAD

**Issue by JonathanFelixBecker** [exislow/tidal-dl-ng#161]
### What happened?

Scenario:
I am trying to download multiple albums.
I am using the latest (v0.15.6) GUI build for macOS ARM.

Bug:
The download proceeds normally until one album download fails.
Then the downloader marks that one as failed (❌) and proceeds to the next album in the list.
When finishing the next album (either because it finished successfully, or it was skipped due to it already existing)
then the downloader repeats that album over and over infinitely (tried for multiple hours).

Example:
-> download album_1 ✅
-> download album_2 ✅
-> download album_3 ✅
-> download album_4 ❌
-> download album_5 ✅
-> download album_5 ✅
-> download album_5 ✅
-> download album_5 ✅
-> ...

Variation:
Sometimes it fails multiple albums in a row.
Then it repeats the first successful download after that.
-> download album_1 ✅
-> download album_2 ✅
-> download album_3 ✅
-> download album_4 ❌
-> download album_5 ❌
-> download album_6 ❌
-> download album_7 ✅
-> download album_7 ✅
-> download album_7 ✅
-> download album_7 ✅
-> ...

Is this a bug everyone is experiencing?

### Version App

v0.15.6

### What operating system are you seeing the problem on?

macOS

### Relevant log output

```shell
> All setup.
........
> Next download will start in 2.3 seconds.
> Next download will start in 3.6 seconds.
> Next download will start in 4.0 seconds.
> Next download will start in 3.2 seconds.
> Next download will start in 4.5 seconds.
> Next download will start in 2.4 seconds.
> Next download will start in 2.4 seconds.
> Next download will start in 4.0 seconds.
> Next download will start in 4.4 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/01 - United In Grief.flac'
> Next download will start in 2.9 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/02 - N95.flac'
> Next download will start in 4.3 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/03 - Worldwide Steppers.flac'
> Next download will start in 4.2 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/04 - Die Hard.flac'
> Next download will start in 2.3 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/05 - Father Time.flac'
> Next download will start in 3.5 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/06 - Rich (Interlude).flac'
> Next download will start in 4.1 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/07 - Rich Spirit.flac'
> Next download will start in 2.5 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/08 - We Cry Together.flac'
> Next download will start in 4.3 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/09 - Purple Hearts.flac'
> Next download will start in 4.2 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/01 - Count Me Out.flac'
> Next download will start in 2.4 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/02 - Crown.flac'
> Next download will start in 4.6 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/03 - Silent Hill.flac'
> Next download will start in 4.2 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/04 - Savior (Interlude).flac'
> Next download will start in 4.3 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/05 - Savior.flac'
> Next download will start in 4.7 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/06 - Auntie Diaries.flac'
> Next download will start in 4.6 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/07 - Mr. Morale.flac'
> Next download will start in 3.0 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/08 - Mother I Sober.flac'
> Next download will start in 3.3 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/09 - Mirror.flac'
> Next download will start in 2.3 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/01 - The Heart Part 5.flac'
> Next download will start in 4.2 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/01 - United In Grief.flac'
> Next download will start in 3.4 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/02 - N95.flac'
> Next download will start in 4.8 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/03 - Worldwide Steppers.flac'
> Next download will start in 3.2 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/04 - Die Hard.flac'
> Next download will start in 2.2 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/05 - Father Time.flac'
> Next download will start in 4.4 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/06 - Rich (Interlude).flac'
> Next download will start in 4.3 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/07 - Rich Spirit.flac'
> Next download will start in 4.3 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/08 - We Cry Together.flac'
> Next download will start in 4.0 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/09 - Purple Hearts.flac'
> Next download will start in 4.6 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/01 - Count Me Out.flac'
> Next download will start in 2.9 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/02 - Crown.flac'
> Next download will start in 3.5 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/03 - Silent Hill.flac'
> Next download will start in 5.0 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/04 - Savior (Interlude).flac'
> Next download will start in 4.2 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/05 - Savior.flac'
> Next download will start in 4.6 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/06 - Auntie Diaries.flac'
> Next download will start in 3.2 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/07 - Mr. Morale.flac'
> Next download will start in 4.2 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/08 - Mother I Sober.flac'
> Next download will start in 2.6 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/09 - Mirror.flac'
> Next download will start in 3.6 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/01 - The Heart Part 5.flac'
> Next download will start in 4.7 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/01 - United In Grief.flac'
> Next download will start in 4.0 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/02 - N95.flac'
> Next download will start in 3.9 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/03 - Worldwide Steppers.flac'
> Next download will start in 3.4 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/04 - Die Hard.flac'
> Next download will start in 3.8 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/05 - Father Time.flac'
> Next download will start in 4.7 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/06 - Rich (Interlude).flac'
> Next download will start in 3.8 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/07 - Rich Spirit.flac'
> Next download will start in 3.8 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/08 - We Cry Together.flac'
> Next download will start in 3.6 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/09 - Purple Hearts.flac'
> Next download will start in 2.8 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/01 - Count Me Out.flac'
> Next download will start in 4.1 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/02 - Crown.flac'
> Next download will start in 4.3 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/03 - Silent Hill.flac'
> Next download will start in 3.1 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/04 - Savior (Interlude).flac'
> Next download will start in 4.7 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/05 - Savior.flac'
> Next download will start in 3.9 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/06 - Auntie Diaries.flac'
> Next download will start in 4.7 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/07 - Mr. Morale.flac'
> Next download will start in 3.2 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/08 - Mother I Sober.flac'
> Next download will start in 4.8 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/09 - Mirror.flac'
> Next download will start in 2.7 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/01 - The Heart Part 5.flac'
> Next download will start in 4.2 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/01 - United In Grief.flac'
> Next download will start in 4.1 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/02 - N95.flac'
> Next download will start in 4.0 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/03 - Worldwide Steppers.flac'
> Next download will start in 2.9 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/04 - Die Hard.flac'
> Next download will start in 2.7 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/05 - Father Time.flac'
> Next download will start in 2.5 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/06 - Rich (Interlude).flac'
> Next download will start in 3.9 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/07 - Rich Spirit.flac'
> Next download will start in 3.6 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/08 - We Cry Together.flac'
> Next download will start in 2.4 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/09 - Purple Hearts.flac'
> Next download will start in 3.6 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/01 - Count Me Out.flac'
> Next download will start in 3.9 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/02 - Crown.flac'
> Next download will start in 2.1 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/03 - Silent Hill.flac'
> Next download will start in 4.5 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/04 - Savior (Interlude).flac'
> Next download will start in 3.5 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/05 - Savior.flac'
> Next download will start in 2.9 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/06 - Auntie Diaries.flac'
> Next download will start in 3.5 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/07 - Mr. Morale.flac'
> Next download will start in 2.2 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/08 - Mother I Sober.flac'
> Next download will start in 4.5 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/09 - Mirror.flac'
> Next download will start in 4.7 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/01 - The Heart Part 5.flac'
> Next download will start in 3.0 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/01 - United In Grief.flac'
> Next download will start in 3.7 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/02 - N95.flac'
> Next download will start in 3.6 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/03 - Worldwide Steppers.flac'
> Next download will start in 4.2 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/04 - Die Hard.flac'
> Next download will start in 5.0 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/05 - Father Time.flac'
> Next download will start in 3.9 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/06 - Rich (Interlude).flac'
> Next download will start in 2.8 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/07 - Rich Spirit.flac'
> Next download will start in 5.0 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/08 - We Cry Together.flac'
> Next download will start in 2.6 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/09 - Purple Hearts.flac'
> Next download will start in 2.4 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/01 - Count Me Out.flac'
> Next download will start in 2.1 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/02 - Crown.flac'
> Next download will start in 2.9 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/03 - Silent Hill.flac'
> Next download will start in 4.4 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/04 - Savior (Interlude).flac'
> Next download will start in 4.3 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/05 - Savior.flac'
> Next download will start in 3.6 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/06 - Auntie Diaries.flac'
> Next download will start in 4.6 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/07 - Mr. Morale.flac'
> Next download will start in 2.5 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/08 - Mother I Sober.flac'
> Next download will start in 3.9 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/09 - Mirror.flac'
> Next download will start in 4.9 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/01 - The Heart Part 5.flac'
> Next download will start in 2.0 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/01 - United In Grief.flac'
> Next download will start in 2.8 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/02 - N95.flac'
> Next download will start in 4.0 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/03 - Worldwide Steppers.flac'
> Next download will start in 4.2 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/04 - Die Hard.flac'
> Next download will start in 3.6 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/05 - Father Time.flac'
> Next download will start in 3.1 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/06 - Rich (Interlude).flac'
> Next download will start in 3.1 seconds.
> Download skipped, since file exists: '/PATH/Albums/Kendrick Lamar (2022) - Mr. Morale & The Big Steppers/07 - Rich Spirit.flac'
> Next download will start in 2.8 seconds.
.......
```

### Your settings

```json
{"skip_existing": "exact", "lyrics_embed": false, "lyrics_file": true, "video_download": false, "download_delay": true, "download_base_path": "/PATH/TO/LOCATION", "quality_audio": "HI_RES_LOSSLESS", "quality_video": "1080", "format_album": "Albums/{album_artist} ({album_year}) - {album_title}/{album_track_num} - {track_title}", "format_playlist": "Playlists/{playlist_name}/{track_num} - {track_title} - {artist_name} ({album_year}) - {album_title}", "format_mix": "Mix/{mix_name}/{artist_name} - {track_title}", "format_track": "Tracks/{artist_name} - {track_title}{track_explicit}", "format_video": "Videos/{artist_name} - {track_title}{track_explicit}", "video_convert_mp4": false, "path_binary_ffmpeg": "/opt/homebrew/bin/ffmpeg", "metadata_cover_dimension": "1280", "extract_flac": true, "downgrade_on_hi_res": false}
```

**Comment by Kyusetzu** [comment]
I'm experiencing the same issue on Linux Mint, the same TDN version.

To add to the problem:

Even without duplicate filtering, it just keeps redownloading the files over and over in a loop.

**Comment by exislow** [comment]
Thanks for reporting this issue. Please be more precise to avoid confusion.

> Bug:
> The download proceeds normally until one album download fails.
> Then the downloader marks that one as failed (❌) and proceeds to the next album in the list.
> When finishing the next album (either because it finished successfully, or it was skipped due to it already existing)
> then the downloader repeats that album over and over infinitely (tried for multiple hours).

Can you exactly specify what album is meant by "that album" in your example?

> Variation:
> Sometimes it fails multiple albums in a row.
> Then it repeats the first successful download after that.

Same here. What album(s) would it be according to your example?

**Comment by loddar66** [comment]
"that album" is always the 1st one which gets downloaded successfully after a failure. Then you have to quit the program and restart it, adding all downloads from the scratch. I could also explain it in German if you like..

I think you would be able to reproduce it if you found an album with a track lasting just a few seconds - which always leads to an error...

**Comment by exislow** [comment]
Got it, thanks. Can you maybe provide such an album, where you know for sure, that the download fails and the whole thing gets stuck?

**Comment by loddar66** [comment]
I posted an example in #187 - If I remember correctly, this caused the queue processing to stop working when trying to download in album mode...

https://tidal.com/browse/album/305339144

**Comment by loddar66** [comment]
P.S.: trying to download an arbitrary album (in HI_RES) in the same session (after one 5 second track from the album mentioned above caused an issue) ends up in the endless loop again...

**Comment by exislow** [comment]
Thanks for your help. it is fixed with https://github.com/exislow/tidal-dl-ng/commit/600c2dda68b9b099c4335f481fe6d6af3f8e66b4

**Comment by loddar66** [comment]
> Thanks for your help. it is fixed with [600c2dd](https://github.com/exislow/tidal-dl-ng/commit/600c2dda68b9b099c4335f481fe6d6af3f8e66b4)

You're welcome. Now I'm looking forward to trying out a fixed release any day soon...😉
